nilearn/_utils/estimator_checks.py

```diff
@@ -589,6 +590,13 @@ def check_masker_fitted(estimator):
     # so we can pass nifti image to surface maskers.
     with pytest.raises(ValueError, match=_not_fitted_error_message(estimator)):
         estimator.transform(_img_3d_rand())
+    with pytest.raises(ValueError, match=_not_fitted_error_message(estimator)):
+        estimator.transform_single_imgs(_img_3d_rand())
+    if is_multimasker(estimator):
+        with pytest.raises(
+            ValueError, match=_not_fitted_error_message(estimator)
+        ):
+            estimator.transform_imgs([_img_3d_rand()])
```

Comment:
unrelated but spotted in this PR

had a check to make that all transform related public methods run check_is_fitted before running